docs/events/wg_workshops/index.md

```diff
@@ -9,4 +9,34 @@ rsecon22/index
 
 ```
 
-The working group has agreed to run quarterly workshops to discuss project progress, latest TRE developments, and actions for the community to take together. This page will record reports and outputs from each of these events.
+The community has agreed to run quarterly workshops to discuss project progress, latest TRE developments, and actions for the community to take together. This page will record reports and outputs from each of these events.
+
+## Background
+
+The community workshops happen four times a year, with 3 sessions fully online, half-day events, and one hybrid (primarily in-person) event once a year.
+The in-person event so far has aligned nicely with RSE Con each year, however as the community builds and develops this may be updated.
+
+### Aims
+
+The aims of the meeting are ...
+
+1. **Bring the community together**: There are many people working in the TRE space in the UK and abroad, facing similar issues and problems when building this complex critical infrastructure. The primary aim of these meetings is to bring this community together to learn from each other, and move forwards together.
+2. **Share knowledge**: Through our primary presentations, workshops and breakout discussions, these meetings provide the space for the community to learn about the TRE landscape, latest TRE developments, and what work others are doing in the space.
+3. **Collaboratively solve problems**: These meetings are a chance for the community to bring forward problems they are facing day-to-day in their TRE work, and resolve these issues in a group setting. This helps promote the benefits of knowledge sharing and community engagement in achieving success.
+4. **Identify and advance community work**: As well as learning about individual project teams' work, these meetings are a chance to hear about our working groups, and identify the work we want to take forwards as a community.
+
+### Structure
+
+The meetings vary slightly from session to session, however they broadly follow this structure:
+
+#### Half day meetings
+
+- **Introduction & Working Group updates**: An introduction to the day, followed by updates from each of the working groups on what they have done since the last meeting
+- **'Keynote' presentation**: A central talk for all attendees to hear about a key aspect of the TRE landscape and have a group discussion. Previous talks have included introductions to DARE UK, the NHS SNSDE work, and OpenSAFELY
```

Comment:
OpenSAFELY: Are you planning to have this merged _after_ the next meeting?

Reply:
Doesn't really matter it's only in a few weeks